.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,34 @@
+name: Python package
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+        python-version: [3.8]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip setuptools
+          pip install --upgrade --force-reinstall --no-cache-dir --editable=".[cpu,dev]"
+          pip install --no-cache-dir "cloudpickle==1.3.0"
```

Comment:
I know the travis.yml file had this, but it shouldn't be necessary anymore.

Reply:
@kaczmarj - you mean the cloudpickle part?

Reply:
```suggestion
```